scripts/gendocs/generate-new-import-path-docs

```diff
@@ -158,7 +152,7 @@ LATEST_VERSION="docs" # to also upgrade a specific version, use v1.13-docs, v1.1
 #genversionwithcli "release-1.12" "v1.12-docs"
 #genversionwithcli "release-1.13" "v1.13-docs"
 
-genversionwithcli "master" "$LATEST_VERSION"
+genversionwithcli "release-1.14" "$LATEST_VERSION"
```

Comment:
I changed the default branch so that it generates exactly the right CLI docs and API docs.
Can be reverted after we release v1.14.0.